src/constructors.jl

```diff
@@ -53,6 +53,8 @@ function Taylor1(x::T, order::Int) where {T<:Number}
     v[1] = x
     return Taylor1(v, order)
 end
+Taylor1(x::T) where {T<:Number} = Taylor1(x, 0)
```

Comment:
I think this method introduces an ambiguity with other existing methods which are often used, in particular with `Taylor1(5)`, which defines a `Taylor1{Float64}` independent variable or order 5. I would therefore propose not to include this method.

Reply:
Thanks! I removed that method to avoid ambiguity

Reply:
Thanks!